plugins/flashrom/meson.build

```diff
@@ -1,4 +1,4 @@
-if get_option('plugin_flashrom')
+if get_option('plugin_flashrom') or get_option('plugin_tuxec')
```

Comment:
although this is functional - I don't think it's the right thing to do.  If someone compiles with flashrom turned off but tuxec they'll still get flashrom.  What you should do is add a check in tuxec/meson.build that flashrom is turned on or off, and report an error if flashrom is off but tuxec is on.